src/main.jsx

```diff
@@ -3,8 +3,4 @@ import ReactDOM from 'react-dom/client'
 import App from './containers/App'
 import './styles/main.scss'
 
-ReactDOM.createRoot(document.getElementById("root")).render(
-  <StrictMode>
-    <App />
-  </StrictMode>
-);
+ReactDOM.createRoot(document.getElementById('root')).render(<App />)
```

Comment:
```suggestion
ReactDOM.createRoot(document.getElementById("root")).render(
  <StrictMode>
    <App />
  </StrictMode>
);
```

plus, check comment for src/containers/NewProducts/NewProducts.jsx